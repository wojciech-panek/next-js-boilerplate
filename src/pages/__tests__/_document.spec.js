import React from 'react';
import { shallow } from 'enzyme';

import Document from '../_document';


describe('Document: Page', () => {
  const defaultProps = {
  };

  const component = (props) => (
    <Document {...defaultProps} {...props} />
  );

  it('should render correctly', () => {
    const wrapper = shallow(component());
    global.expect(wrapper).toMatchSnapshot();
  });
});
