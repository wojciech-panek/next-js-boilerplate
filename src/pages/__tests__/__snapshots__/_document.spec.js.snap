// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document: Page should render correctly 1`] = `
<html>
  <Head>
    <meta
      content="width=device-width, initial-scale=1, shrink-to-fit=no"
      key="viewport"
      name="viewport"
    />
    <link
      href="/static/icons/favicon.ico"
      rel="shortcut icon"
    />
    <link
      href="/static/icons/apple-touch-icon.png"
      rel="apple-touch-icon"
      sizes="180x180"
    />
    <link
      href="/static/icons/favicon-32x32.png"
      rel="icon"
      sizes="32x32"
      type="image/png"
    />
    <link
      href="/static/icons/favicon-16x16.png"
      rel="icon"
      sizes="16x16"
      type="image/png"
    />
    <link
      color="#5bbad5"
      href="/static/icons/safari-pinned-tab.svg"
      rel="mask-icon"
    />
    <meta
      content="#ffffff"
      name="theme-color"
    />
    <meta
      content="yes"
      name="mobile-web-app-capable"
    />
    <link
      href="/static/manifest.json"
      rel="manifest"
    />
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</html>
`;
